webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -289,7 +289,7 @@ class VolumeTracingController @Inject()(
           // consecutive 3D points (i.e., nine floats) form a triangle.
           // There are no shared vertices between triangles.
           tracing <- tracingService.find(tracingId) ?~> Messages("tracing.notFound")
-          (vertices, neighbors) <- if (tracing.mappingIsEditable.getOrElse(false))
+          (vertices, neighbors) <- if (tracing.hasEditableMapping.getOrElse(false))
```

Comment:
`getHasEditableMapping` could be used here as well afaik
```suggestion
          (vertices, neighbors) <- if (tracing.getHasEditableMapping)
```